tests/www/views/test_views_dagrun.py

```diff
@@ -162,7 +162,7 @@ def test_set_dag_runs_action(
         follow_redirects=True,
     )
     check_content_in_response(expected_message, resp)
-    assert [ti.state for ti in session.query(TaskInstance).all()] == expected_ti_states
+    assert {ti.state for ti in session.query(TaskInstance).all()} == set(expected_ti_states)
```

Comment:
Does change parametrize parameter `expected_ti_states ` are the better choose? For `clear` test, change `expected_ti_states` from `[None, None]` to `{None}`

Reply:
Yeah good call, thanks.